src/responder/pam/pamsrv_passkey.c

```diff
@@ -993,6 +994,23 @@ pam_passkey_auth_send(TALLOC_CTX *mem_ctx,
     return req;
 }
 
+static void
+passkey_child_timeout(struct tevent_context *ev,
+                      struct tevent_timer *te,
+                      struct timeval tv, void *pvt)
+{
+    struct tevent_req *req =
+            talloc_get_type(pvt, struct tevent_req);
+    struct pam_passkey_auth_send_state *state =
+            tevent_req_data(req, struct pam_passkey_auth_send_state);
+
+    DEBUG(SSSDBG_CRIT_FAILURE, "Timeout reached for passkey child\n");
```

Comment:
Hi,

since there is the `passkey_child_timeout` option, what do you think about mention it here, e.g. similar to `p11_child_timeout()`.

bye,
Sumit

Reply:
Agreed, updated.